data/en/getencoding.json

```diff
@@ -3,7 +3,7 @@
 	"type":"function",
 	"syntax":"getEncoding(scope_name)",
 	"returns":"string",
-	"related":[],
+	"related":["cfcontent", "cfprocessingdirective", "URLDecode", "URLEncodedFormat"],
```

Comment:
I don't think URLDecode or URLEncodedFormat should be related here. This function deals with character set, not URL encoding.